real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java

```diff
@@ -0,0 +1,105 @@
+package com.roboclub.robobuggy.main;
+
+import gnu.io.CommPortIdentifier;
+
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+import com.roboclub.robobuggy.logging.RobotLogger;
+import com.roboclub.robobuggy.simulation.SensorPlayer;
+import com.roboclub.robobuggy.ui.Gui;
+
+
+/** This class is the driver starting up the robobuggy program, if you want the buggy to drive itself you should run this node */
+public class RobobuggyMainFile {
+    static Robot buggy;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1779952891/issues/source?bid=2938665&fileBranchId=3059924#l16)